bentoctl/operator/constants.py

```diff
@@ -1,15 +1,16 @@
 YATAI_OPERATOR_NAME = 'yatai'
 
-OFFICIAL_OPERATORS = {
-    "yatai": "bentoml/yatai-deployment-operator",
-    "heroku": "bentoml/heroku-deploy",
-    "aws-lambda": "bentoml/aws-lambda-deploy",
-    "aws-sagemaker": "bentoml/aws-sagemaker-deploy",
-    "aws-ec2": "bentoml/aws-ec2-deploy",
-    "azure-functions": "bentoml/azure-functions-deploy",
-    "azure-container-instances": "bentoml/azure-container-instances-deploy",
-    "google-compute-engine": "bentoml/google-compute-engine-deploy",
-    "google-cloud-run": "bentoml/google-cloud-run-deploy",
-}
 
 MAIN_BRANCH = "main"
```

Comment:
should we lock to a certain hash for each bentoctl release?

Reply:
> 

probably after we don't have a lot of updates. Not right now